src/Microsoft.ML.Ensemble/OutputCombiners/MultiVoting.cs

```diff
@@ -33,7 +33,7 @@ private static VersionInfo GetVersionInfo()
                 loaderAssemblyName: typeof(MultiVoting).Assembly.FullName);
         }
 
-        private sealed class Arguments : ArgumentsBase
```

Comment:
>Arguments [](start = 29, length = 9)

Even if it's private would be good to rename this one too.

Reply:
i am not touching the private ones in this PR..  (i think i mentioned this in my reply to the other comment)


---
In reply to: [257413177](https://github.com/dotnet/machinelearning/pull/2563#discussion_r257413177) [](ancestors = 257413177)